index.js

```diff
@@ -21,23 +21,24 @@ class RunTask {
         this.runOne(taskItem, data, eachDone);
       }, done);
     }
-    const fn = this.tasks[task];
+    let fn = this.tasks[task];
+    if (this.options.bind) {
```

Comment:
this logic should be moved down after the array check, then you don't need the function check